### DISCUSSION TITLE
404 Illustrations (and other fun 404 things)

### DISCUSSION THREAD

**Discussion by raae** [raae/nattermob.dev#3] [Ideas]
On stream today there was a lot of great input on the 404 page!

Keep them coming in this thread.

**Comment by olavea** [comment]
AK (https://github.com/AgataMaria) mentioned LEGO kraken in the chat on the live stream
https://youtu.be/lD7i1jGBeMk

So I wrote this to AK on twitter (https://twitter.com/ja_gatka/status/1380799132445577217)
Totally unrelated, but when you mentioned LEGO kraken for the 404 page in the chat on the live stream yesterday.
https://youtu.be/lD7i1jGBeMk
 Did you have any have anything specific in mind? I mean have you seen a @LEGOIdeas 🐙 somewhere?
Please answer here:

**Comment by olavea** [comment]
AK's First Mate made this LEGO portrait of her:


![AK_EynglP2WYAQvBfW](https://user-images.githubusercontent.com/7642493/114273266-ad764000-9a19-11eb-931f-6571dbdfc1a5.jpeg)


https://pbs.twimg.com/media/EynglP2WYAQvBfW?format=jpg&name=small

**Comment by olavea** [comment]
Kraken colored and drawn by Lillian Raae-Vea (6 🦄 ) 







![Kraken_Lillian_Ola_1 1](https://user-images.githubusercontent.com/7642493/114914165-480fbe00-9e22-11eb-9e99-047ca9f2f31c.jpeg)


Kraken colored by Lillian Raae-Vea (6 🦄 ) drawn by Ola Vea

![Kraken_Lillian_Ola_1](https://user-images.githubusercontent.com/7642493/114913847-f49d7000-9e21-11eb-80be-adeea560beb6.jpeg)
